index.html

```diff
@@ -3,6 +3,7 @@
   <head>
     <meta charset="UTF-8" />
     <link rel="icon" type="image/svg+xml" href="/vite.svg" />
+    <link rel="stylesheet" href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/4.7.0/css/font-awesome.min.css">
```

Comment:
Maybe we could include the style sheet locally so this doesn't break if they're offline? Mod manager should still work offline, just would show nothing in the database I imagine. 

https://www.npmjs.com/package/font-awesome

Reply:
Good point, will have to test how its fetch reacts to failure